hops-leader-election/src/main/java/io/hops/leaderElection/LeaderElection.java

```diff
@@ -110,7 +110,7 @@ public void run() {
             "sucessfulTx " + sucessfulTx + " failedTx " + failedtx +
             " time period " + context.time_period + " " + te.getMessage(), te);
         // transaction failed
-        sleepDuration = 0;
+        sleepDuration = 50;
```

Comment:
why is this one 50 and the next one 1000?

Reply:
For transient exceptions, such as deadlocks, it is ok to retry immediately. For non-transient exceptions wait for a little longer. changed "0" to 50 so that the thread does not hog the cpu